airbyte-integrations/connectors/source-it-glue/unit_tests/test_streams.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from http import HTTPStatus
+from unittest.mock import MagicMock
+
+import pytest
+from source_it_glue.source import ItGlueStream
+
+# from urllib import response
+
+
+@pytest.fixture()
+def config(request):
+    args = {"api_key": "YXNmnhkjf", "fatId": "12345"}
+    return args
+
+
+@pytest.fixture
+def patch_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(ItGlueStream, "path", "v0/example_endpoint")
+    mocker.patch.object(ItGlueStream, "primary_key", "test_primary_key")
+    mocker.patch.object(ItGlueStream, "__abstractmethods__", set())
+
+
+def test_request_params(patch_base_class, config):
+    stream = ItGlueStream(**config)
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": None}
+    expected_params = {"page[size]": 1000}
+    assert stream.request_params(**inputs) == expected_params
+
+
+# def test_next_page_token(patch_base_class,config):
+#     stream = ItGlueStream(**config)
+#     response = MagicMock()
+#     response.json.return_value  = {"next-page":{"https://api.itglue.com/expirations?page[size]=1000&page[number]=2"}}
+#     inputs = {"response" : response}
+#     expected_token = {"page[size]": ["1000"], "page[number]": ["2"]}
+#     assert stream.next_page_token(**inputs) == expected_token
```

Comment:
any reason why this is commented?

Reply:
I will update and remove all the commented lines from the connector.